packages/perseus/src/widgets/radio/base-radio.tsx

```diff
@@ -383,15 +382,13 @@ const BaseRadio = function (props: Props): React.ReactElement {
                     return (
                         <li
                             key={i}
-                            // @ts-expect-error - TS2322 - Type 'HTMLLIElement | null' is not assignable to type 'null'.
                             ref={(e) => (listElem = e)}
-                            // @ts-expect-error - TS2322 - Type 'readonly string[]' is not assignable to type 'string'.
                             className={className}
-                            // @ts-expect-error - TS2322 - Type 'null' is not assignable to type 'MouseEventHandler<HTMLLIElement> | undefined'.
                             onClick={clickHandler}
-                            // @ts-expect-error - TS2322 - Type '((e: TouchEvent) => void) | null' is not assignable to type 'TouchEventHandler<HTMLLIElement> | undefined'.
                             onTouchStart={
-                                !labelWrap ? null : captureScratchpadTouchStart
+                                labelWrap
```

Comment:
The React event handlers are defined as `React.TouchEvent<T> | undefined`.  If we assign `null`, then we have to suppress Typescript. So changing to use `undefined` instead of `null` fixes that.